draft-kohbrok-mimi-portability.md

```diff
@@ -86,21 +86,25 @@ Client                    Source Delivery Service           Target Delivery Serv
 
 TODO: The MLS group ID is fixed for the lifetime of a group and cannot be
 changed. This requires group IDs to be globally unique across all relevant
-Delivery Services. One way to achiev this is to ensure a group ID is composed of
-a locally unique part (per Delivery Service) and cobined with FQDN of the
-Delivery Service. This needs to be described in more detail.
+Delivery Services. Even though this could be solved using a UUID, groups should
+also be tied to a specific owning delivery service. For the purpose of this
+document, we solve this by including an MLS GroupContext Extension in every
+group that contains the FQDN of the group's designated DS. This needs to be
+described in more detail.
```

Comment:
For now, don't use an extension. Then instead of a proposal, just put the information into the AAD.